pkg/systemd/services.html

```diff
@@ -168,6 +171,17 @@
     </div>
   </script>
 
+  <script id="service-empty-tmpl" type="x-template/mustache">
+    <div id="empty-search" class="blank-slate-pf blank-screen">
+      <h1 translate>
+        No Matching Results
```

Comment:
The whitiespacing doesn't work, it gets copied literally to cockpit.pot:

    msgid "        No Matching Results      "

Please write it as `<h1>Bla</h1>`.

Reply:
Ach, did not notice that. Thank you for pointing this out. Repushed